agent/hibinit-agent

```diff
@@ -118,78 +118,87 @@ def find_grub_mount():
                  '/etc/grub2.cfg']
     for ln in path_list:
         if os.path.isfile(ln) and os.access(ln, os.R_OK):
-            cmd = 'stat -L -c %m ' + ln
-            mount = check_output(cmd, shell=True, universal_newlines=True)
+            find_mount_cmd = 'stat -L -c %m ' + ln
+            mount = check_output(find_mount_cmd, shell=True, universal_newlines=True)
             return mount.strip()
     return None
 
 
 def patch_grub_config(swap_device, offset):
-    log("Updating GRUB to use the device %s with offset %d for resume" % (swap_device, offset))
-    cmd = "grubby --update-kernel=ALL --args='no_console_suspend={console} resume_offset={offset} resume={swap_device}'"
-    cmd = cmd.format(console='1', offset=offset, swap_device=swap_device)
     mount_point = find_grub_mount()
+
     if mount_point is None:
-        log("Grub configuration is not updated. Grub cannot found under /boot or /etc. Please run manual grub update with resume parameters")
+        print_to_sys_log("Grub configuration is not updated. Grub cannot found under /boot or /etc. " +
+                         "Please run manual grub update with resume parameters")
         return
-    fsfreeze = "sync && mountpoint -q {mount} && trap '' HUP INT QUIT TERM && fsfreeze -f {mount} && fsfreeze -u {mount}"
-    fsfreeze = fsfreeze.format(mount=mount_point)
-    mkconfig = "grub2-mkconfig -o {path}"
-    mkconfig = mkconfig.format(path="/boot/grub2/grub.cfg")
-    check_output(mkconfig, shell=True)
-    check_call(cmd, shell=True)
-    check_call(fsfreeze, shell=True)
-
-    # Some of grubby versions need a restart after changing in kernel parameters
+
```

Comment:
Should we refactor this into separate method for mkconfig_cmd , grub_update_kernel_cmd and  fsfreeze_cmd

Reply:
I suppose this is reasonable, but I did want to avoid creating too many functions as the whole agent is written in a single file. 

Balancing a few consideration:
* These commands will only be used a single time, so separate methods would not reduce duplicate code
* The `cmd` definitions & execution were already reordered for better legibility

Am open to the suggestion so up for further discussion

Reply:
I think here there is a scope to refactor it to make the code more readable.

Reply:
I believe legibility is sufficient without dissecting this function into multiple sub-functions. 

However in this case, adding the suffix `_cmd` seems largely superfluous and actual hinders legibility. 

Renaming these without said suffix makes this section of code much more legible.